### ISSUE TITLE
Can reduce the Redundantancy in the codes

### ISSUE THREAD

**Issue by jhonsnow456** [TusharAMD/Runn#16]

**Comment by jhonsnow456** [comment]
Hi @TusharAMD can i solve this issue I found few redundant code which I can be refactored through a function.

**Comment by TusharAMD** [comment]
Yes sure I will assign the issue but can you please show which exact part will be modified? Also after editing please run the program and make sure its running like before. Please add screenshots too.

**Comment by jhonsnow456** [comment]
Yeah sure I will add it to the review only if the code works and you can see it through file change tabs.

**Comment by TusharAMD** [comment]
Ok go ahead.
